fulp_modules/features/toys/code/toys.dm

```diff
@@ -24,3 +24,12 @@
 	attack_verb_continuous = list("dust")
 	attack_verb_simple = list("dust")
 	squeak_override = list('sound/effects/supermatter.ogg'= 1)
+
+/obj/item/toy/plush/pico
+	name = "pico plushie"
+	desc = "A plushie of an alarmed middle schooler with orange hair and a green T-Shirt. Maybe something's happening at his school."
+	icon = 'fulp_modules/features/toys/icons/toys.dmi'
+	icon_state = "pico"
+	attack_verb_continuous = list("flails at")
+	attack_verb_simple = list("flail at")
+	squeak_override = list('fulp_modules/features/toys/sound/pico.ogg'= 1)
```

Comment:
I'd give him another sound, since the one you used is the classroom's ambience, y'know, apples.

Reply:
Actually never mind, watched the video you attached to the PR's body. Sounds good.